src/connectedk8s/azext_connectedk8s/vendored_sdks/operations/_connected_cluster_operations.py

```diff
@@ -461,12 +462,19 @@ def list_cluster_user_credential(
         accept = "application/json"
 
         # Construct URL
-        url = self.list_cluster_user_credential.metadata['url']  # type: ignore
+        if namespace is None:
```

Comment:
Please confirm if the file is generated automatically using autorest and it's not changed manually